lib/fathead/npm_cli/cover/redirects.txt

```diff
@@ -0,0 +1,74 @@
+access public,access
+access restricted,access
+access grant,access
```

Comment:
@rahulxxarora it looks like these are all taken care of now in your `output.txt`? You can remove this file, or ensure it only contains redirects that aren't being generated by your script